package.json

```diff
@@ -36,7 +36,7 @@
     "prepare": "husky install",
     "sense": "nebula sense",
     "spec": "scriptappy-from-jsdoc -c ./scriptappy.config.js",
-    "start": "nebula serve  --type sn-pivot-table",
+    "start": "yarn run locale:generate && nebula serve  --type sn-pivot-table",
```

Comment:
do we need it here?

Reply:
Maybe, but I guess it depends on how you use the scripts.

If, for example, the first thing you do when having cloned the repo, is `yarn` and than `yarn start` we want it to contain the generated locale.

I myself don't use the build scripts as `yarn start` do the watch and building as well. But if I changed a locale file, it probably run the generate script myself.

Reply:
oh I didn't know start does building as well, then it makes sense